src/NuGet.Services.AzureSearch/Models/SearchDocument.cs

```diff
@@ -19,6 +19,7 @@ public static class SearchDocument
         [SerializePropertyNamesAsCamelCase]
         public class Full : AddFirst
         {
+            [IsFilterable]
             public long? TotalDownloadCount { get; set; }
```

Comment:
What's this for?

Reply:
the scoring profile can only be added to a field that is "filterable"